ncm-nss/src/test/perl/configure.t

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+# -*- mode: cperl -*-
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Quattor qw(simple);
+use NCM::Component::sysctl;
```

Comment:
probably not what you want (current works because `Test::MockModule` below takes care of loading nss module)